addons/mail/static/src/components/notification_list/notification_list.js

```diff
@@ -37,13 +38,7 @@ class NotificationList extends Component {
      */
     get notifications() {
         const { notifications } = this.storeProps;
-        return notifications.map(notification =>
-            notification.type === 'thread'
-                ? Object.assign({}, notification, {
-                    thread: this.env.models['mail.thread'].get(notification.uniqueId),
-                })
-                : notification
-        );
```

Comment:
I'm not against the change, but what was this code and why can it be removed?

Reply:
See https://github.com/odoo-dev/odoo/pull/265#discussion_r438691968